pocs/camera/canon_gphoto2.py

```diff
@@ -33,24 +33,29 @@ def connect(self):
         if _serial_number > '':
             self._serial_number = _serial_number
 
-        self.set_property('/main/actions/viewfinder', 1)       # Screen off
-        self.set_property('/main/settings/autopoweroff', 0)     # Don't power off
-        self.set_property('/main/settings/reviewtime', 0)       # Screen off
-        self.set_property('/main/settings/capturetarget', 0)    # Internal RAM (for download)
-        self.set_property('/main/settings/artist', 'Project PANOPTES')
-        self.set_property('/main/settings/ownername', 'Project PANOPTES')
-        self.set_property('/main/settings/copyright', 'Project PANOPTES 2016')
-        self.set_property('/main/imgsettings/imageformat', 9)       # RAW
-        self.set_property('/main/imgsettings/imageformatsd', 9)     # RAW
-        self.set_property('/main/imgsettings/imageformatcf', 9)     # RAW
-        self.set_property('/main/imgsettings/iso', 1)               # ISO 100
-        self.set_property('/main/capturesettings/focusmode', 0)         # Manual
-        self.set_property('/main/capturesettings/continuousaf', 0)         # No AF
-        self.set_property('/main/capturesettings/autoexposuremode', 3)  # 3 - Manual; 4 - Bulb
-        self.set_property('/main/capturesettings/drivemode', 0)         # Single exposure
-        self.set_property('/main/capturesettings/shutterspeed', 0)      # Bulb
-        # self.set_property('/main/actions/syncdatetime', 1)  # Sync date and time to computer
-        # self.set_property('/main/actions/uilock', 1)        # Don't let the UI change
+        # Properties to be set upon init.
+        properties = {
+            '/main/actions/viewfinder': 1,
+            '/main/settings/autopoweroff': 0,
+            '/main/settings/reviewtime': 0,
+            '/main/settings/capturetarget': 0,
+            '/main/imgsettings/imageformat': 9,
+            '/main/imgsettings/imageformatsd': 9,
+            '/main/imgsettings/imageformatcf': 9,
+            '/main/imgsettings/iso': 9,
+            '/main/capturesettings/focusmode': 0,
+            '/main/capturesettings/continuousaf': 0,
+            '/main/capturesettings/autoexposuremode': 3,
+            '/main/capturesettings/drivemode': 0,
+            '/main/capturesettings/shutterspeed': 0,
+        }
+
+        self.set_property_values(properties)
+
+        if first_init:
```

Comment:
Why not include these 3 properties with the ones above?

Reply:
Because they need to be set with `--set-config-value` (rather than `-index`) so I was going to separate them and then build another bulk set by value method but it seemed a bit overkill. Ideally I think we want some kind of one-time setup of hardware script that would run this as it is a little redundant to run each time. Not sure yet what the best way to do that is.

Reply:
I think just setting them each time is fine, esp. if we have a way to do it "relatively" fast. Perhaps a method that takes two maps, one with property to index, and the other with property to value? Then they can all be done in a single call.